engine/include/cubos/engine/plugins/tools/scene_editor.hpp

```diff
@@ -0,0 +1,48 @@
+#ifndef CUBOS_ENGINE_PLUGINS_SCENE_EDITOR_HPP
+#define CUBOS_ENGINE_PLUGINS_SCENE_EDITOR_HPP
+
+#include <cubos/engine/cubos.hpp>
+#include <string>
+
+using namespace cubos::engine;
+
+namespace cubos::engine::plugins::tools
+{
+    struct SceneInfo;
+
+    using entity_map_t = std::map<std::string, cubos::core::ecs::Entity>;
+    using scene_map_t = std::map<std::string, SceneInfo>;
```

Comment:
I think its better to just use the map type instead of the alias.
It clutters the namespace + also we don't use this naming scheme.